src/officehours_api/tests.py

```diff
@@ -136,33 +136,28 @@ def test_first_meeting_removal_bad_phone_logs_exception(self, mock_twilio: mock.
         self.assertEqual(self.exceptions, 1)
 
     @mock.patch('officehours_api.notifications.twilio')
-    def test_first_meeting_removal_doesnt_notify_optout(self, mock_twilio: mock.MagicMock):
-        m1 = self.create_meeting([self.foo,])
-        self.create_meeting([self.bar, self.baz, self.attendeeoptout,])
-        mock_twilio.reset_mock()
-        m1.delete()
+    def test_meeting_start_doesnt_notify_optout(self, mock_twilio: mock.MagicMock):
+        self.create_meeting([self.bar, self.baz, self.attendeeoptout,], True)
         receivers = self.get_receivers(mock_twilio)
         self.assertFalse(
             receivers >=
             {'+15555550003',}
         )
 
     @mock.patch('officehours_api.notifications.twilio')
-    def test_second_meeting_removal_notifies_none(self, mock_twilio: mock.MagicMock):
-        self.create_meeting([self.foo,])
-        m2 = self.create_meeting([self.bar, self.baz])
-        mock_twilio.reset_mock()
-        m2.delete()
+    def test_meeting_start_doesnt_notify_other(self, mock_twilio: mock.MagicMock):
+        m1 = self.create_meeting([self.foo,])
+        self.create_meeting([self.baz,])
+        m1.start(self.hostie)
+        m1.save()
         receivers = self.get_receivers(mock_twilio)
-        self.assertFalse('+15555550001' in receivers)
         self.assertFalse('+15555550002' in receivers)
 
     @mock.patch('officehours_api.notifications.twilio')
-    def test_first_meeting_removal_doesnt_notify_second_in_line(self, mock_twilio: mock.MagicMock):
-        m1 = self.create_meeting([self.foo,])
-        self.create_meeting([self.bar,])
-        self.create_meeting([self.baz,])
-        mock_twilio.reset_mock()
-        m1.delete()
+    def test_meeting_start_notifies_attendees(self, mock_twilio: mock.MagicMock):
+        self.create_meeting([self.foo, self.bar,], True)
         receivers = self.get_receivers(mock_twilio)
-        self.assertFalse('+15555550002' in receivers)
+        self.assertTrue(
+            receivers >=
```

Comment:
I'm forgetting now, what's the thinking behind using `>=` over strict equality? I may be not understanding something about set comparison.

Reply:
This is confirming that those values are "in" the set. It could allow there to be more values. Not sure if that even makes a practical difference in this instance, but I chose it because the name of the test implies we're asserting that the attendees were notified (`...notifies_attendees`)